infrastructure/base/prometheus/release.yaml

```diff
@@ -17,10 +17,6 @@ spec:
     - kind: ConfigMap
       name: cms-prometheus-values
       valuesKey: values.yaml
-    - kind: ConfigMap
```

Comment:
Why are you removing this? We have these settings in cm-prod-promteheus. They may be gone from the int branch, but production still relies on them

Reply:
Ah, I did not check. Simply cherry-picked from int_globus branch, since the changes were all in base. 
Let me check.

Reply:
I see now. Removed the commit with these changes. 
Ready for review.